models/Post.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+
+const PostSchema = new mongoose.Schema({
```

Comment:
I think leaving short description for code will be good for improving the code quality like " // This is Schema for POST model"